hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/HdfsScheduler.java

```diff
@@ -30,6 +30,7 @@
 import java.util.Arrays;
 import java.util.List;
 import java.util.Observable;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 
```

Comment:
I'd rather not suppress warnings here and instead write it in our standard style as required by gradle.